src/app/src/components/SubmissionDetail.js

```diff
@@ -0,0 +1,153 @@
+                                    {label}
+                                </Text>
+                                {/* if not a string, accept a React element */}
+                                {entry instanceof String ? (
+                                    <Text
+                                        mt={1}
+                                        textStyle='submissionDetailBody'
+                                    >
+                                        {entry}
+                                    </Text>
+                                ) : (
+                                    entry
+                                )}
+                            </GridItem>
+                        );
+                    })}
+                </Grid>
+            </>
+        );
+    };
+
+    return (
+        <Flex h='100vh' paddingTop={10} paddingBottom={10} bg='gray.50'>
+            <Box ml={8} mr={8} w={650}>
+                <Flex>
+                    <IconButton
+                        icon={<Icon as={ArrowLeftIcon} />}
+                        aria-label='Back'
+                        mr={6}
+                        onClick={() => navigate('/submissions')}
+                    />
+                    <Heading size='lg' mr={6}>
+                        {submissionName}
+                    </Heading>
+                    <SubmittedBadge variant='submissionDetail' />
+                </Flex>
+                {makeDataGrid({
+                    initialMarginTop: 7,
+                    title: 'Primary contact',
+                    data: {
+                        'Full name': contactName,
+                        'Phone number': contactPhone,
+                        'Job title': contactTitle,
+                    },
+                })}
+                {makeDataGrid({
+                    initialMarginTop: 10,
+                    title: 'Water system information',
+                    data: {
+                        PWSID: pwsId,
+                        'Water system name': utilityName,
+                        'Mailing address': (
+                            <>
+                                <Text mt={1} textStyle='submissionDetailBody'>
+                                    {utilityAddress1}
+                                </Text>
+                                <Text mt={1} textStyle='submissionDetailBody'>
+                                    {utilityAddress2}
+                                </Text>
+                                <Text mt={1} textStyle='submissionDetailBody'>
+                                    {cityStateZip}
+                                </Text>
+                            </>
+                        ),
+                    },
+                })}
+                <Box>
+                    <Checkbox isChecked={sharingPreference} mt={6} mb={12}>
+                        I would like to share data with surrounding towns
+                    </Checkbox>
+                </Box>
```

Comment:
Sorry this was not removed in the Wireframes, but let's get rid of this. This is no longer an option for the contributors, will be set at the State level.